_posts/2020-05-19-pathfinding-devlog.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Pathfinding for Enemy chase behavior, Difficulty with 2D navmashes
+description: Containing instructions for writing up posts for the development log.
+---
+
+# Development Log Dated 05-23-2020
```

Comment:
One nitpick -- don't post-date the date in the filename (per Pisan's instructions); I agree that it should be 5/23, just change the filename on this in another commit.

Reply:
I meant to write 23 but I was using your devlog as a template on how to format so my brain pulled a funny and wrote 19.